src/app/services/features.ts

```diff
@@ -9,5 +10,8 @@ export const features = (env: Env['var']): Features => {
         return true
       }
     },
+    'use service-insights': (_can, mesh: Mesh) => {
```

Comment:
very interesting naming. Is this a good practice to use spaces in object members?

Reply:
In javascript it doesn't matter really, especially not in this case.

The reason for this is so the usage of these things reads nicely, we have a function called `can` (which you can see in this PR) that reads like "if you can use service-insights":

```js
if(can('use service-insights')) {

}
```

There are others like `create zones` i.e. `can('create zones') ? showCreateButton : dontShowCreateButton`